project/Dependencies.scala

```diff
@@ -17,7 +17,7 @@ object Dependencies {
     val annotationApi           = "1.3.2"
     val cats                    = "2.6.1"
     val mockito                 = "5.11.0"
-    val munit                   = "1.0.0-M8"
+    val munit                   = "1.0.0-M11"
```

Comment:
```suggestion
    val munit                   = "1.0.0-M12"
```
we updated to newer version already, M11 might be broken.

Reply:
:ok_hand: done.